src/runtime_src/core/pcie/tools/xbmgmt/cmd_config.cpp

```diff
@@ -28,13 +28,13 @@
 const char *subCmdConfigDesc = "Parse or update daemon/device configuration";
 
 const char *subCmdConfigUsage =
-    "--device [--card bdf] --runtime_clk_scale en(dis)able\n"
+    "--device [--card bdf] [--runtime_clk_scale (en/dis)able] [--cs_threshold_power_override val] [--cs_threshold_temp_override val] [--cs_reset val]\n"
     "--enable_retention [--ddr] [--card bdf]\n"
     "--disable_retention [--ddr] [--card bdf]\n";
 const char *subCmdConfigExpUsage =
     "Experts only:\n"
     "--daemon --host ip-or-hostname-for-peer\n"
-    "--device [--card bdf] [--security level] [--cs_threshold_power_override val] [--cs_reset val]\n"
+    "--device [--card bdf] [--security level] [--cs_threshold_power_override val] [--cs_threshold_temp_override val] [--cs_reset val]\n"
```

Comment:
if an option is already displayed to the user, it doesn't make sense to add it to experts only command as well. Please remove `[--cs_threshold_power_override val] [--cs_threshold_temp_override val] [--cs_reset val]`

Has this been signed off by PRT? I remember that these options got moved to experts only for some reason. Maybe Max can confirm